app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -8,7 +8,7 @@ def callback
 
     session[:user_info] = auth.extra.raw_info
 
-    redirect_to claim_path(journey: policy.routing_name, slug: "teacher-detail")
+    redirect_to claim_path(journey: Journeys.for_policy(policy)::ROUTING_NAME, slug: "teacher-detail")
```

Comment:
I think we might have some circular dependency going on here; I spotted a new failure in the pipeline yesterday; just FYI, I'll look into that when I get a chance; unrelated to this PR anyway

Reply:
Yes, I thought it was gone thanks to your changes, but I did get another related failure yesterday, and after rebasing, see [here](https://dfe-ssp.visualstudio.com/S118-Teacher-Payments-Service/_build/results?buildId=455097&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&s=6884a131-87da-5381-61f3-d7acc3b91d76&t=68dc271a-6b90-554f-dc73-9843e30016f8&l=8892)

Reply:
Thanks, yes I could see how that could happen if `claim_from_session` returned `nil`, which it shouldn't do in practice, but it would be worth fixing this.